wiki/Tournaments/4DM/2/en.md

```diff
@@ -59,12 +59,69 @@ The 4 Digit osu!mania World Cup 2 is run by various community members.
 - [Map suggestion form](https://forms.gle/BjLjtUSAVDLhhQ368)
 - **[Statistics sheet](https://docs.google.com/spreadsheets/d/1cT1BLBuG3XGyDvPWFTGJO_TOpRkdKQaH6shstbJcauc/edit?usp=sharing)**
 
+---
+
 ## Participants
 
-*The participants will be revealed after the qualifiers ends.*
+|  | Country | Members |
```

Comment:
could put the flag and country name in same column. the reason there was this extra column for flags in some other tournaments was to keep things aligned when teams were formed from multiple countries